products/schema.json

```diff
@@ -0,0 +1,483 @@
+            "properties": {
+              "github_releases": {
+                "title": "Github releases",
+                "description": "Github repo to track the releases from in the format {author}/{repo}.",
+                "type": "string"
+              },
+              "regex": {
+                "$ref": "#/$defs/regex"
+              }
+            },
+            "required": [
+              "github_releases"
+            ],
+            "additionalProperties": false
+          },
+          {
+            "type": "object",
+            "properties": {
+              "pypi": {
+                "title": "Pypi",
+                "description": "Package name on pypi.",
+                "type": "string"
+              },
+              "regex": {
+                "$ref": "#/$defs/regex"
+              },
+              "template": {
+                "$ref": "#/$defs/template"
+              }
+            },
+            "required": [
+              "pypi"
+            ],
+            "additionalProperties": false
+          },
+          {
+            "type": "object",
+            "properties": {
+              "maven": {
+                "title": "Maven",
+                "description": "Package name on maven.",
+                "type": "string"
+              },
+              "regex": {
+                "$ref": "#/$defs/regex"
+              },
+              "template": {
+                "$ref": "#/$defs/template"
+              }
+            },
+            "required": [
+              "maven"
+            ],
+            "additionalProperties": false
+          }
+        ]
+      }
+    },
+    "releases": {
+      "title": "Releases",
+      "description": "Array of releases for this product.",
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/release"
+      }
+    }
+  },
+  "required": [
+    "title",
+    "category",
+    "permalink",
+    "releases"
+  ],
+  "$defs": {
+    "boolOrDate": {
+      "oneOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "boolean"
+        }
+      ]
+    },
+    "link": {
+      "type": "string",
+      "pattern": "^/"
+    },
+    "template": {
+      "title": "Template",
+      "description": "Template which will be used to create the final release.",
+      "type": "string"
+    },
+    "regex": {
+      "title": "Regex",
+      "description": "Regex which will be used to filter the releases.",
+      "type": "string"
+    },
+    "release": {
+      "type": "object",
+      "properties": {
+        "releaseCycle": {
+          "title": "releaseCycle",
+          "description": "The release cycle which this release is part of.",
+          "examples": [
+            "1",
+            "1.0"
+          ],
+          "type": "string"
+        },
+        "link": {
+          "title": "Link",
+          "description": "Link to changelog for the latest release.",
+          "type": "string",
```

Comment:
Should also accept `None` values (=> `null` or no value).

Reply:
fixed in 88284c1